tyk-pro/scripts/bootstrap.sh

```diff
@@ -13,6 +13,9 @@ main(){
   then
     syntax_message
   else
+    # Sleep for 10 seconds to allow for dashboard to come up.
+    sleep 10
```

Comment:
Better to check for the deployment status to be deterministic about this, also Helm's `--wait` should be doing this already.

Reply:
I am going to replace this with retires on the /org endpoint. --wait is currently not resolving the issue. If you want to recreate the issue disable gateway installation on helm. You will run into the problem then. Retries are much more elegant however.